install/recyclarr-install.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y tzdata
+$STD apt-get install -y git
+$STD apt-get install -y tini
+msg_ok "Installed Dependencies"
+
+var_recyclarr_url="https://github.com/recyclarr/recyclarr/releases/latest/download/recyclarr-linux-x64.tar.xz"
+var_app_dir="/opt/recyclarr"
+var_app_file="$var_app_dir/recyclarr"
+var_config_file="$var_app_dir/recyclarr.yml"
+var_recyclarr_cron_file="$var_app_dir/recyclarr.cron"
+
+msg_info "Installing Alpine-Recyclarr"
+mkdir -p "$var_app_dir"
+chmod 775 "$var_app_dir"
+wget "$var_recyclarr_url" -O - | tar xJ --overwrite -C "$var_app_dir"
+msg_ok "Installed Alpine-Recyclarr"
+
+msg_info "Configuring Alpine-Recyclarr"
+chmod +x "$var_app_file"
+export PATH="${PATH}:$var_app_dir"
+recyclarr config create --path "$var_config_file"
```

Comment:
There's some potential here to add a better default configuration file based on user inputs. `recyclarr create` supports a `--template` argument for specifying what template to start out with. The downside is that the available templates might change and that means maintenance. Maybe it's fine to leave it be and just add some documentation on where to find the templates.

https://recyclarr.dev/wiki/cli/config/create/